mycroft/api/__init__.py

```diff
@@ -424,6 +426,29 @@ def upload_skills_data(self, data):
             "json": to_send
             })
 
+    def upload_wake_word(self, audio, params):
+        """Helper method for sending files using the device API"""
```

Comment:
Nitpick: The `audio` and `params` parameters should be documented in the docstring.